code-of-conduct.rst

```diff
@@ -33,14 +33,14 @@ Encouraged Behaviour
 
 Examples of encouraged behaviour include:
 
-- Respectful discourse of disagreements around new features, changes, and
+- Discuss and disagree respectfully concerning new features, changes, and
```

Comment:
I think that's changing the original meaning which is that when there are disagreements, they occur respectfully. Open to other wording choices but I don't think debate is the same.